common_operations/force_merge.json

```diff
@@ -0,0 +1,25 @@
+{
+    "operation": {
+        "operation-type": "force-merge",
+        "request-timeout": 7200{%- if max_num_segments is defined %},
```

Comment:
Do you feel `request_timeout` should be parameterized here, so someone who wants to override it can do it?  It's unlikely, but you never know.

Reply:
Personally, I think there's no harm in parameterizing most of these values, and just having a default value like:
```
"request-timeout": {{ request_timeout | default(60) | tojson }}{%- if max_num_segments is defined %},
```

I updated the PR with this change. Users can pass in a custom request-timeout value with:
```
        {% with 
         default_index_settings={},
         request-timeout=X
        %}
        {{ benchmark.collect(parts="../../common_operations/workload_setup.json") }},
        {% endwith %}
```

Reply:
I'm of the opinion that there's no harm in parameterizing most of the workload values. I added a param for request-timeout in the latest revision. Let me know what you think!